_data/other.yml

```diff
@@ -335,3 +335,10 @@ websites:
       software: Yes
       sms: Yes
       doc: https://support.zendesk.com/hc/en-us/articles/203824246
+
+    - name: Toodledo
+      url: https://www.toodledo.com
+      img: toodledo.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.toodledo.com/otp.php
```

Comment:
for the `doc`, what about https://www.toodledo.com/info/help.php#168? It's not behind a sign-in page